rero_ils/modules/loans/api.py

```diff
@@ -395,6 +395,17 @@ def get_overdue_loans():
     return overdue_loans
 
 
+def search_loans_for_item(item_pid):
```

Comment:
Why not having a function that returns the most recent transaction date?

Reply:
updated.